cmd/lifecycle/extender.go

```diff
@@ -60,6 +60,7 @@ func (e *extendCmd) Exec() error {
 	extender, err := extenderFactory.NewExtender(
 		e.AnalyzedPath,
 		e.AppDir,
+		fmt.Sprint(e.UID),
```

Comment:
This UID is what the extender is running as (or rather, what it will drop privileges to once extension is finished), which _should_ be the CNB_USER_ID from the builder image, but is not guaranteed to be. Also when extending run images the runtime user may be different from the build time user.

Reply:
I think if we're always just reading from the config then we don't need the new field, right?